sw/airborne/subsystems/ahrs/ahrs_float_cmpl.c

```diff
@@ -208,324 +208,325 @@ void ahrs_fc_update_accel(struct FloatVect3 *accel, float dt)
+    struct FloatVect3 residual_imu;
+    float_rmat_vmult(&residual_imu, &ahrs_fc.ltp_to_imu_rmat, &residual_ltp);
 
 
-  /* Complementary filter proportional gain.
-   * Kp = 2 * zeta * omega * weight * ahrs_fc.mag_cnt
-   * with ahrs_fc.mag_cnt beeing the number of propagations since last update
-   */
-  const float mag_rate_update_gain = 2 * ahrs_fc.mag_zeta * ahrs_fc.mag_omega * ahrs_fc.mag_cnt;
-  RATES_ADD_SCALED_VECT(ahrs_fc.rate_correction, residual_imu, mag_rate_update_gain);
+    /* Complementary filter proportional gain.
+     * Kp = 2 * zeta * omega * weight * ahrs_fc.mag_cnt
+     * with ahrs_fc.mag_cnt beeing the number of propagations since last update
+     */
+    const float mag_rate_update_gain = 2 * ahrs_fc.mag_zeta * ahrs_fc.mag_omega * ahrs_fc.mag_cnt;
+    RATES_ADD_SCALED_VECT(ahrs_fc.rate_correction, residual_imu, mag_rate_update_gain);
 
-  /* Complementary filter integral gain
-   * Correct the gyro bias.
-   * Ki = (omega*weight)^2 * dt
-   */
-  const float mag_bias_update_gain = -(ahrs_fc.mag_omega * ahrs_fc.mag_omega) * dt;
-  RATES_ADD_SCALED_VECT(ahrs_fc.gyro_bias, residual_imu, mag_bias_update_gain);
-}
+    /* Complementary filter integral gain
+     * Correct the gyro bias.
+     * Ki = (omega*weight)^2 * dt
+     */
+    const float mag_bias_update_gain = -(ahrs_fc.mag_omega * ahrs_fc.mag_omega) * dt;
+    RATES_ADD_SCALED_VECT(ahrs_fc.gyro_bias, residual_imu, mag_bias_update_gain);
+  }
 
 
-void ahrs_fc_update_mag_2d_dumb(struct FloatVect3 *mag)
-{
+  void ahrs_fc_update_mag_2d_dumb(struct FloatVect3 *mag)
+  {
 
-  /* project mag on local tangeant plane */
-  struct FloatEulers ltp_to_imu_euler;
-  float_eulers_of_rmat(&ltp_to_imu_euler, &ahrs_fc.ltp_to_imu_rmat);
-
-  const float cphi   = cosf(ltp_to_imu_euler.phi);
-  const float sphi   = sinf(ltp_to_imu_euler.phi);
-  const float ctheta = cosf(ltp_to_imu_euler.theta);
-  const float stheta = sinf(ltp_to_imu_euler.theta);
-  const float mn = ctheta * mag->x + sphi * stheta * mag->y + cphi * stheta * mag->z;
-  const float me =     0. * mag->x + cphi          * mag->y - sphi          * mag->z;
-
-  const float res_norm = -RMAT_ELMT(ahrs_fc.ltp_to_imu_rmat, 0, 0) * me +
-                         RMAT_ELMT(ahrs_fc.ltp_to_imu_rmat, 1, 0) * mn;
-  const struct FloatVect3 r2 = {RMAT_ELMT(ahrs_fc.ltp_to_imu_rmat, 2, 0),
-          RMAT_ELMT(ahrs_fc.ltp_to_imu_rmat, 2, 1),
-          RMAT_ELMT(ahrs_fc.ltp_to_imu_rmat, 2, 2)
-  };
-  const float mag_rate_update_gain = 2.5;
-  RATES_ADD_SCALED_VECT(ahrs_fc.rate_correction, r2, (mag_rate_update_gain * res_norm));
-  const float mag_bias_update_gain = -2.5e-4;
-  RATES_ADD_SCALED_VECT(ahrs_fc.gyro_bias, r2, (mag_bias_update_gain * res_norm));
+    /* project mag on local tangeant plane */
+    struct FloatEulers ltp_to_imu_euler;
+    float_eulers_of_rmat(&ltp_to_imu_euler, &ahrs_fc.ltp_to_imu_rmat);
 
-}
+    const float cphi   = cosf(ltp_to_imu_euler.phi);
```

Comment:
what is the reason for changes here (they aren't mentioned in the PR description)?

Reply:
Weird, all this is done by fix_code_style. I only did the two minor changes in commit 8cdc6d7057618a0a5bf4735bcc7a63e40ce927e4

Reply:
Ok makes sense